Makefile

```diff
@@ -284,6 +284,8 @@ check-contrib:
 	@$(MAKE) -C $(CONTRIB_PATH) for-all CMD="$(GOCMD) mod edit -replace go.opentelemetry.io/collector/config/configtls=$(CURDIR)/config/configtls"
 	@$(MAKE) -C $(CONTRIB_PATH) for-all CMD="$(GOCMD) mod edit -replace go.opentelemetry.io/collector/config/internal=$(CURDIR)/config/internal"
 	@$(MAKE) -C $(CONTRIB_PATH) for-all CMD="$(GOCMD) mod edit -replace go.opentelemetry.io/collector/confmap=$(CURDIR)/confmap"
+	@$(MAKE) -C $(CONTRIB_PATH) for-all CMD="$(GOCMD) mod edit -replace go.opentelemetry.io/collector/connector=$(CURDIR)/connector"
+	@$(MAKE) -C $(CONTRIB_PATH) for-all CMD="$(GOCMD) mod edit -replace go.opentelemetry.io/collector/connector/forwardconnector=$(CURDIR)/connector/forwardconnector"
```

Comment:
Hmmm... this looks incorrect

Reply:
I'm not sure why this change is appearing here.  It's already been made on `main` and was brought into this branch by merging main into the source branch of this PR via #7944.

Reply:
Possibly because of the squash merge? It might be easier to close this pr and trigger a new one :(

Reply:
I'm not sure that's necessary, but I can do it if you'd prefer.  It looks like this will still end up with the same end-state.